Source/WebInspectorUI/UserInterface/Views/MediaDetailsSidebarPanel.js

```diff
@@ -0,0 +1,319 @@
+        this.#ui.channelsRow = new MediaDetailsSectionSimpleRow(WI.UIString("Channels", "Channels @ Media Sidebar", "Title for Channels row in Media Sidebar"));
+
+        let audioGroup = new WI.DetailsSectionGroup([this.#ui.audioCodecRow, this.#ui.sampleRateRow, this.#ui.channelsRow]);
+        this.#ui.audioSection = new WI.DetailsSection("media-audio-details", WI.UIString("Audio Details", "Audio Details @ Media Sidebar", "Title for Audio Details section in Media Sidebar"), [audioGroup]);
+    }
+
+    layout()
+    {
+        this.#ui.sourceRow.updateValue();
+        this.#ui.viewportRow.updateValue();
+        this.#ui.framesRow.updateValue();
+        this.#ui.resolutionRow.updateValue();
+        this.#ui.videoFormatRow.updateValue();
+        this.#ui.audioFormatRow.updateValue();
+        this.#ui.videoCodecRow.updateValue();
+        this.#ui.transferRow.updateValue();
+        this.#ui.primariesRow.updateValue();
+        this.#ui.matrixRow.updateValue();
+        this.#ui.fullRangeRow.updateValue();
+        this.#ui.audioCodecRow.updateValue();
+        this.#ui.sampleRateRow.updateValue();
+        this.#ui.channelsRow.updateValue();
+    }
+
+    // DOMDetailsSidebarPanel Overrides
+
+    supportsDOMNode(nodeToInspect)
+    {
+        return nodeToInspect.isMediaElement();
+    }
+
+    attached()
+    {
+        super.attached();
+        this.#startUpdateTimer();
+    }
+
+    detached()
+    {
+        super.detached();
+        this.#cancelUpdateTimer();
+    }
+
+    // Protected
+
+    initialLayout()
+    {
+        super.initialLayout();
+
+        this.element.appendChild(this.#ui.generalSection.element);
+        this.element.appendChild(this.#ui.videoSection.element);
+        this.element.appendChild(this.#ui.audioSection.element);
+    }
+
+    // Private
+
+    #values = {
+        source: null,
+        viewport: null,
+        devicePixelRatio: null,
+        quality: null,
+        videoFormat: null,
+        audioFormat: null,
+        video: null,
+        audio: null
+    };
+
+    #ui = {
+        sourceRow: null,
+        viewportRow: null,
+        framesRow: null,
+        resolutionRow: null,
+        videoFormatRow: null,
+        audioFormatRow: null,
+        generalSection: null,
+        videoCodecRow: null,
+        transferRow: null,
+        primariesRow: null,
+        matrixRow: null,
+        fullRangeRow: null,
+        videoSection: null,
+        audioCodecRow: null,
+        sampleRateRow: null,
+        channelsRow: null,
+        audioSection: null
+    };
+
+    #updateTimer = null;
+    #updateTimerInterval = 250;
+
+    #startUpdateTimer()
+    {
+        this.#cancelUpdateTimer();
+        this.#updateTimerFired();
+    }
+
+    #cancelUpdateTimer()
+    {
+        if (this.#updateTimer)
+            clearTimeout(this.#updateTimer);
```

Comment:
I _think_ it's safe to call `clearTimeout` with a null value, it'll just not do anything.